docs/pages/kubernetes-access/getting-started/local.mdx

```diff
@@ -0,0 +1,349 @@
+
+</Notice>
+
+## Step 2/4 Create a user
+
+We will create a Teleport user who can access Kubernetes Dashboard from
+outside the cluster. To do this, we will use the `tctl` administrative tool from
+inside the pod that runs the Auth Service and Proxy Service.
+
+Run the following commands to get the name of the relevant pod and execute a
+`tctl` command to create a user:
+
+```code
+# We will use this variable for subsequent commands
+$ PROXY_POD=$(kubectl get po -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+$ kubectl exec $PROXY_POD -- tctl users add --roles=access appuser
+User "appuser" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+https://teleport-cluster.teleport-cluster.svc.cluster.local:443/web/invite/<token>
+```
+
+Next, open a browser at the URL returned by `tctl users add` command. minikube's
+reverse tunnel will allow you to access the Teleport Proxy Service at
+`https://teleport-cluster.teleport-cluster.svc.cluster.local:443`.
+
+(!docs/pages/includes/insecure-certificate.mdx!)
+
+![Teleport User Registration](../../../img/quickstart/login.png)
+
+In the Teleport Web UI, enter a password and scan the QR code with your OTP
+application to create your user.
+
+<Details title="Don't want to use MFA in your local demo?" opened={false}>
+We recommend requiring a second factor for all Teleport users. However, for
+convenience while setting up your local demo, you can run the following command
+to create a dynamic configuration resource that disables MFA for your demo user:
+
+```code
+$ kubectl exec -i $PROXY_POD -- bash -c "cat<<EOF>>/home/cp.yaml
+kind: cluster_auth_preference
+version: v2
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: local
+  second_factor: 'off'
+EOF
+tctl create --force --confirm /home/cp.yaml" 
+```
+</Details>
+
+After signing in, you will navigate to the Teleport Web UI.
+
+
+## Step 3/4 Enable access to Kubernetes Dashboard
+
+We will configure the Teleport Application Service to proxy traffic to
+Kubernetes Dashboard so we can access it securely from outside the cluster.
+
+To do so, we will generate a token that our Application Service Node can use to
+register itself with the cluster, then run the Application Service with the
+token, configuring it to access Kubernetes Dashboard.
+
+Run the following commands to generate the token, which is specific to
+Kubernetes Dashboard:
+
+```code
+# The cluster IP of Kubernetes Dashboard
+$ DASH_ADDR=$(kubectl -n kubernetes-dashboard get service kubernetes-dashboard -o jsonpath="{.spec.clusterIP}")
+$ kubectl exec -i $PROXY_POD -- tctl tokens add \
+--type=app \
+--app-name=kube-dash \
+--app-uri=https://$DASH_ADDR
+The invite token: <token>.
+This token will expire in 60 minutes.
+...
+```
+
+Copy the join token so you can assign it to `JOIN_TOKEN` below, then launch the
```

Comment:
Since we use the terminology `invite token` in the code block above, I suggest we stick with that here.

```suggestion
Copy the invite token so you can assign it to `JOIN_TOKEN` below, then launch the
```